src/users/users.spec.ts

```diff
@@ -219,7 +219,15 @@ describe('UserManagement', () => {
           email_address: 'bob.loblaw@example.com',
         })
         .reply(200, {
-          id: 'auth_challenge_01E4ZCR3C56J083X43JQXF3JK5',
+          user: {
+            object: 'user',
+            email: 'test01@example.com',
+            firstName: 'Test 01',
+            lastName: 'User',
+            emailVerified: true,
```

Comment:
The shape of this response isn't correct, as the API will return snake_cased values:

```suggestion
            email_verified: true,
```